app/src/views/private/private-view.vue

```diff
@@ -557,8 +569,9 @@ const showLicenseBanner = computed(() => userStore.isAdmin && settingsStore.sett
 		}
 
 		&.is-open {
-			transform: translateX(0);
+			transform: translateX(0) !important;
 		}
```

Comment:
Let’s prevent `!important`.

```suggestion
		html &.is-open {
			transform: translateX(0);
		}
```